docs/advisors/checks/mongodb-psa-architecture.md

```diff
@@ -0,0 +1,36 @@
+# MongoDB PSA Architecture
+
+## Description
+This check returns an error if the replicaSet is using a PSA architecture.
+
+## Resolution
+
+A replica set can be configured with either a PSS (Primary-Secondary-Secondary) or a PSA (Primary-Secondary-Arbiter) architecture.
+
+In a three-member replica set with a primary-secondary-arbiter (PSA) architecture or a sharded cluster with three-member PSA shards, the arbiter does not hold a copy of the data. These deployments provide only one complete copy of the data. They are used primarily for situations where nodes are located in different Data Centers and you want to preserve quorum in cases where you want to avoid split brain caused by potential network issues.
+
+Arbiters require fewer resources and can thus provide lower cost for the cluster. However, using a PSA architecture reduces the redundancy and fault tolerance that MongoDB has built-in natively. It is not recommended to use PSA (Primary-Secondary-Arbiter) architecture in Production environments due to below **serious concerns.**
+
+**Data Redundancy**
+
+**Fault Tolerance**
+
+If a Secondary node is down in a PSA architecture, the replica set is left  with only one Primary & one arbiter.  Since an Arbiter is not a full data-bearing node, this causes some serious operational consequences listed below:
```

Comment:
```suggestion
If a secondary node is down in a PSA architecture, the replica set is left  with only one primary & one arbiter.  Since an arbiter is not a full data-bearing node, this causes some serious operational consequences listed below:
```